tfhe/src/integer/bigint/algorithms.rs

```diff
@@ -53,7 +55,7 @@ pub(crate) fn leading_zeros(lhs: &[u64]) -> u32 {
 }
 
 // Order of words must be little endian
-pub(crate) fn compare<T>(lhs: T, rhs: T) -> std::cmp::Ordering
+pub(crate) fn compare_unsigned<T>(lhs: T, rhs: T) -> std::cmp::Ordering
 where
     T: AsRef<[u64]>,
```

Comment:
possible to have UnsignedNumeric as a bound ?

Reply:
We would need to find a way to add an associated  to Signed/Unsigned Numeric without breaking existing things

Also I think this would make the calling this part from the `signed_comparison` harder.

Reply:
ok, if you find something then good otherwise we'll live with it